R/print.R

```diff
@@ -0,0 +1,87 @@
+#' Print method for `<multi_epidist>` class
+#'
+#' @param x A `<multi_epidist>` object.
+#' @param n A `numeric` specifying how many `<epidist>` objects to print. This
+#' argument is passed to [head()] for `list` printing. Default is `NULL` and the
+#' number of elements to print is controlled by package [options()].
+#' @inheritParams print.epidist
+#'
+#' @return Invisibly returns a `<multi_epidist>`. Called for side-effects.
+#' @export
+print.multi_epidist <- function(x, ..., n = NULL) {
```

Comment:
```suggestion
print.multi_epidist <- function(x, n = NULL, ...) {
```

Or do you prefer to always force `n` to be named?

Reply:
I think it would be better to force explicit argument naming. Again following the design from {tibble}: https://tibble.tidyverse.org/reference/formatting.html.